shared/src/main/scala/mlscript/JSBackend.scala

```diff
@@ -159,12 +159,12 @@ abstract class JSBackend(allowUnresolvedSymbols: Bool) {
   protected def translateApp(term: App)(implicit scope: Scope): JSExpr = term match {
     // Binary expressions
     case App(App(Var(op), Tup((N -> Fld(_, lhs)) :: Nil)), Tup((N -> Fld(_, rhs)) :: Nil))
-        if JSBinary.operators contains op =>
-      JSBinary(op, translateTerm(lhs), translateTerm(rhs))
+        if JSBinary.operators contains toJSOperator(op) =>
```

Comment:
I think this case is only meant happen in the old parser. It should probably be left alone.

Reply:
In fact, better guard it with a `if !newDefs` as well!